portfolio/src/main/webapp/index.html

```diff
@@ -40,13 +46,14 @@ <h2> Fun Facts</h2>
         I was able to impersonate their calls and birds from the wild. I was known to go to talent shows and just 
         surprise people with my bird calls. 
        </p>
-
-       <img src="images/Blackbeak.jpg">
-       <img src="images/Max.jpeg">
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+     <div id="pets"> 
+        <img src="images/Blackbeak.jpg">
+        <img src="images/Max.jpeg">   
+    </div>
+    <p>Click <a href="/hello">here</a> to see content created by server code.</p>
+    <h3>Click Here To Get A Randomized Song Recommendation From Me!</h3>
+    <button onclick="addRandomSong()">Click Here!</button>
```

Comment:
I know this is just adapting the code you inherited, so feel free to leave this as a TODO, but look into [`addEventListener`](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) to control page functionality purely from your JavaScript, rather than using the `onclick` attribute.

Reply:
I think I'm going to make this into a separate issue. I'm working on it right now an I have an error I'm unsure how to resolve. I think it's best to make this into a separate issue so it's clearer to see what I did.